WeakAurasOptions/SubRegionOptions/LinearProgressTexture.lua

```diff
@@ -0,0 +1,268 @@
+      width = WeakAuras.normalWidth,
+      order = 11,
+      name = L["Re-center X"],
+      min = -0.5,
+      max = 0.5,
+      bigStep = 0.01,
+    },
+    linearTextureUser_y = {
+      type = "range",
+      control = "WeakAurasSpinBox",
+      width = WeakAuras.normalWidth,
+      order = 12,
+      name = L["Re-center Y"],
+      min = -0.5,
+      max = 0.5,
+      bigStep = 0.01,
+    },
+    linearTextureCrop_x = {
+      type = "range",
+      control = "WeakAurasSpinBox",
+      width = WeakAuras.normalWidth,
+      name = L["Crop X"],
+      order = 13,
+      min = 0,
+      softMax = 2,
+      bigStep = 0.01,
+      isPercent = true,
+    },
+    linearTextureCrop_y = {
+      type = "range",
+      control = "WeakAurasSpinBox",
+      width = WeakAuras.normalWidth,
+      name = L["Crop Y"],
+      order = 14,
+      min = 0,
+      softMax = 2,
+      bigStep = 0.01,
+      isPercent = true,
+    },
+    -- Doesn't appear to work
+    linearTextureRotation = {
+      type = "range",
+      control = "WeakAurasSpinBox",
+      width = WeakAuras.normalWidth,
+      name = L["Texture Rotation"],
+      desc = L["Uses Texture Coordinates to rotate the texture."],
+      order = 15,
+      min = 0,
+      max = 360,
+      bigStep = 1
+    },
+    -- Doesn't appear to work
+    linearTextureAuraRotation = {
+      type = "range",
+      control = "WeakAurasSpinBox",
+      width = WeakAuras.normalWidth,
+      name = L["Rotation"],
+      order = 16,
+      min = 0,
+      max = 360,
+      bigStep = 1
+    },
+
+    progress_source = {
+      type = "select",
+      width = WeakAuras.normalWidth,
+      name = L["Progress Source"],
+      order = 17,
+      control = "WeakAurasTwoColumnDropdown",
+      values = OptionsPrivate.Private.GetProgressSourcesForUi(parentData, true),
+      get = function(info)
+        return OptionsPrivate.Private.GetProgressValueConstant(data.progressSources or {-2, ""})
+      end,
+      set = function(info, value)
+        if value then
+          data.progressSources = data.progressSources or {}
+          -- Copy only trigger + property
+          data.progressSources[1] = value[1]
+          data.progressSources[2] = value[2]
+        else
+          data.progressSources = nil
+        end
+        WeakAuras.Add(parentData)
+      end,
+    },
+
+    progress_source_space = {
+      type = "description",
+      name = "",
+      order = 18,
+      width = WeakAuras.normalWidth,
+    },
+
+    anchor_mode = {
+      name = L["Anchor Mode"],
+      type = "select",
+      width = WeakAuras.normalWidth,
+      order = 19,
+      values = OptionsPrivate.Private.anchor_mode,
+    },
+
+    anchor_area = {
+      name = L["Area"],
+      type = "select",
+      width = WeakAuras.normalWidth,
+      control = "WeakAurasTwoColumnDropdown",
+      order = 20,
+      values = areaAnchors,
+      hidden = function()
+        return data.anchor_mode == "point"
+      end
+    },
+
+    self_point = {
```

Comment:
could use a spacer to align anchors & size on same lines
![image](https://github.com/user-attachments/assets/29150659-7bf2-453a-b7dd-a087194b0c59)

btw, something for an other PR, it would be nice if self_point dropdown used WeakAurasAnchorButtons widget, and a new one would have to be created for anchor_point

Reply:
Also could you add x/y offsets?

Reply:
Yeah they make sense for point anchoring.